### DISCUSSION TITLE
Split model to parts with connectors

### DISCUSSION THREAD

**Discussion by xorinzor** [OrcaSlicer/OrcaSlicer#4290] [Q&A]
I need to change a large model to multiple parts, which is easy enough to do.
However, it appears that the only way to add connectors is to use the cut tool, which only uses an entire plane to cut and has no feature to split by parts.

How can I get connectors to these parts easily? I tried meshmixer, but it kept crashing on me and was far from easy to do..
I feel like this really shouldn't be this difficult to achieve, the 2 parts connect only at 1 flat face, I just need the button from the cut tool that allows me to place the connector.

Am I missing something really obvious here?

**Comment by YusufGurel** [comment]
This would really help me too.

**Comment by fabiomolinar** [comment]
Agree on that! Sometimes the models come into multiple parts since the cuts are not always that easy. Being able to simply add connectors to faces without having to cut them would be a cool feature!
